build/pkgs/cachetools/distros/conda.txt

```diff
@@ -0,0 +1 @@
+cachetools
```

Comment:
This is only a dependency of tox, right? Then we don't need to add it - there is no need to micro-manage conda.

Reply:
Recall that for the packages that don't have a conda.txt, bootstrap-conda will emit pip lines.

Reply:
Then this should be modified to be an empty conda file or not?

Reply:
No.

Reply:
?

Reply:
Using empty conda files is something we may have discussed at some point, but it is not our current practice.

Reply:
You are using it yourself in build/pkgs/jupymake/distros/conda.txt

Reply:
Yes, and the comment in this explains its specific use case.

Reply:
So just add a comment here that we a are not actually using cachetools...

Reply:
No, I disagree with this idea.

Reply:
May you care about explaining why, and what advantages you see in installing a package that is not needed

Reply:
There's really not anything conda-specific about the idea that it suffices to only ask for the "top-level" packages because their dependencies are implied via the package manager. But we do not have a mechanism for that, and faking it for the case of conda is not a good approach.

Reply:
Happy to have a solution that works across all distributions... but since you're a modifying Conda files here it would be good enough for me to have a Conda specific solution for now.

Reply:
Let's revisit this idea in another ticket

Reply:
Sure! But this ticket should make sure that the added conda files don't have any negative influences, i.e these packages don't get installed in the default or dev conda environment (which they are not as of now).

 > Recall that for the packages that don't have a conda.txt, bootstrap-conda will emit pip lines.

Only for optional packages though.

Reply:
> these packages don't get installed in the default or dev conda environment (which they are not as of now)

They are not? `tox` is in the generated environment file, so its dependencies are also installed, or am I missing something?

Reply:
Depends, cachetools is but hatch_vcs not. In either way, they are not listed in the environment file, and I don't see any advantages adding them there.

Reply:
You are right about `hatch_vcs`, thanks. 

But the conda package information is also used if one uses @orlitzky's `--enable-system-site-packages` with approach of section 2 ("Using conda to provide system packages for the Sage distribution") of https://doc.sagemath.org/html/en/installation/conda.html

In any case, these build deps are tiny, so I don't think whether they are included in the environment files matters at all.

Reply:
Unless there is an advantage of adding these packages to the environment files, they shouldn't appear there. Might only result in problems such as https://github.com/sagemath/sage/pull/35593#issuecomment-1738092038.

Reply:
> Might only result in problems such as [#35593 (comment)](https://github.com/sagemath/sage/pull/35593#issuecomment-1738092038)

That doesn't seem plausible

Reply:
Well it has to be related to a change in the environment files...

Reply:
Please mark this as unresolved. The purpose of these added conda files is still not clear to me.